### PR TITLE
ROC-4540-added-alert-and-action-group-for-DM-alert

### PR DESCRIPTION
**Pull request by srineelesh**
added alert and action group for DM alert

### CODE REVIEW

File: action-groups.tf [hunk 1]
```diff
@@ -66,3 +66,26 @@ module "cmc-ff4j-admissions-fail-action-group" {
...
+
+  resourcegroup_name = "${azurerm_resource_group.rg.name}"
+  action_group_name = "Document Management Failure Alert - ${var.env}"
+  short_name = "DMF_alert"
+  email_receiver_name = "Document Management Failure Alerts"
+  email_receiver_address = "${data.azurerm_key_vault_secret.document_management_email_secret.value}"
+}
+
+output "document_management_failure_action_group_name" {
+  value = "${module.cmc-bulk-print-fail-action-group.action_group_name}"
```
Comment (andrewwa-kainos) [inline action-groups.tf]: Replace with dm version

File: alerts.tf [hunk 2]
```diff
@@ -58,4 +58,22 @@ module "cmc-ff4j-admissions-fail-alert" {
...
   resourcegroup_name = "${azurerm_resource_group.rg.name}"
+}
+
+module "cmc-document-management-fail-alert" {
+  source = "git@github.com:hmcts/cnp-module-metric-alert"
+  location = "${azurerm_application_insights.appinsights.location}"
+  app_insights_name = "${azurerm_application_insights.appinsights.name}"
+
+  alert_name = "Document Management failure - CMC"
+  alert_desc = "Triggers when a Document Management upload or download failure event is recieved from CMC in a 5 minute poll."
```
Comment (andrewwa-kainos) [inline alerts.tf]: Typo on recieved -> received

Comment (srineelesh) [reply alerts.tf]: well spotted..done..

**Review by andrewwa-kainos — APPROVED** [review]

**Review by tharacka — APPROVED** [review]
LGTM

**Pull request merged**